bittensor/_neuron/text/advanced_server/nucleus_impl.py

```diff
@@ -87,7 +87,54 @@ def __init__(self,
         
         # -- keeps track of gradients applied
         self.backward_gradients = 0 
-        
+        self.set_fine_tuning_params()
+
+    def set_fine_tuning_params(self):
+        r''' Set to tune only the parameter of the last layer
+        '''
```

Comment:
Can you add a test for this?

Reply:
hey i have added some test to the file tests/unit_tests/bittensor_tests/test_neuron.py, please check:)